src/windows/list.cpp

```diff
@@ -148,21 +148,26 @@ bool IsVirtualHardDrive(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
   return false;
 }
 
-bool IsSystemDevice(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
+bool IsSystemDevice(HDEVINFO hDeviceInfo, DeviceDescriptor *device) {
   PWSTR windowsPath = NULL;
+  BOOL isSystem = false;
   HRESULT result = SHGetKnownFolderPath(
     FOLDERID_Windows, 0, NULL, &windowsPath);
```

Comment:
As I mentioned at https://github.com/resin-io-modules/drivelist/pull/243/files#r159242773 should we also check against other common "known folder paths" ? Since some people might have their ProgramFiles or UserProfile on a disk that isn't the same as the Windows disk.

Reply:
Yup, added!